File: SofiForce.Models/Models/EntityModels/AppUserLocationModel.cs
using System;

namespace Models
{
    public class AppUserLocationModel
    {
        public long TrackingId { get; set; }
        public int? UserId { get; set; }
        public int? TrackingTypeId { get; set; }
        public DateTime? TrackingDate { get; set; }
        public DateTime? TrackingTime { get; set; }
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public bool? IsPositive { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/AppUserLocationModel.ts
import { AppUserModel } from "./appUserModel";

export interface AppUserLocationModel {
    trackingId: number;
    userId: number | null;
    trackingDate: Date | null;
    trackingTime: string | null;
    latitude: number | null;
    longitude: number | null;
    trackMode: number | null;

}